File: BMI v.3.0/app/src/main/java/pl/arisa/bmicalculator/model/Quiz.java
package pl.arisa.bmicalculator.model;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class Quiz implements Serializable {
    private final static long serialVersionUID = 2230778689516013709L;

    private List<Question> questions = new ArrayList<>();
    private int correctCounter;

    public List<Question> getQuestions() {
        return questions;
    }

    public void setQuestions(List<Question> questions) {
        this.questions = questions;
    }

    public int getCorrectCounter() {
        return correctCounter;
    }

    public void setCorrectCounter(int correctCounter) {
        this.correctCounter = correctCounter;
    }
}

File: BMI v.4.0/app/src/main/java/pl/arisa/bmicalculator/model/Quiz.kt
package pl.arisa.bmicalculator.model

import java.io.Serializable
import java.util.*

class Quiz : Serializable {
    var questions: List<Question> = ArrayList()
    var correctCounter = 0

    companion object {
        private const val serialVersionUID = 2230778689516013709L
    }
}